src/components/appointments/CreateAppointment.stories.js

```diff
@@ -15,13 +17,19 @@ import {
   mockAppointmentTimesGet,
 } from './mocks';
 
+const withClearSessionStorage = Story => {
+  window.sessionStorage.removeItem(SESSION_STORAGE_KEY);
+  return <Story />;
+};
+
```

Comment:
Discussed: Possibly make it into a global decorator and add a comment on when it runs